web/webkit/src/main/scala/net/liftweb/http/LiftSession.scala

```diff
@@ -2372,7 +2395,20 @@ class LiftSession(private[http] val _contextPath: String, val underlyingId: Stri
     }
   }
 
-  private def buildAndStoreComet[T <: LiftCometActor](newCometFn: (CometCreationInfo)=>Box[T])(creationInfo: CometCreationInfo): Box[T] = {
+  /**
+   * Need to add your own CometActor to a Session/Page? You might 'cause creating
+   * a CometActor programmatically within a snippet may be between that referring
+   * to a CometActor by class name.
+   *
+   * This is dangerous stuff, so use with caution.
+   *
+   * @param newCometFn a Function that takes `CometCreateInfo` and returns a CometActor
+   *                   instance
+   * @param creationInfo the info to send to the above function
+   * @tparam T A LiftCometActor or better
+   * @return The created, registered, etc. CometActor
+   */
+  def buildAndStoreComet[T <: LiftCometActor](newCometFn: (CometCreationInfo)=>Box[T])(creationInfo: CometCreationInfo): Box[T] = {
```

Comment:
Okay... here's an example https://github.com/dpp/devoxx_pl_2015_scala_clojure/blob/alt_comet/src/main/scala/sb/snippet/Actorize.scala#L29

I wound up extending `serverActorForClient` to include the lifecycle call-backs... but I liked the idea of being able to create an arbitrary CometActor class and maybe close over local scope rather than do the wonky "send a message and put the variables from the message in `var`s" paradigm. So I left `buildAndStoreComet` public.